easybuild/easyconfigs/g/GitPython/GitPython-1.0.2.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'Bundle'
+
+name = 'GitPython'
+version = '1.0.2'
+
+homepage = 'https://gitpython.readthedocs.org/en/stable/'
+description = """GitPython is a python library used to interact with git repositories, high-level like git-porcelain, or low-level like git-plumbing."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+exts_filter = ('python -c"import %(ext_name)s"', '')
+
+# allow use of system Python
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+exts_list = [
+    ('setuptools', '18.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/setuptools/'],
+    }),
+    ('pip', '7.1.0', {
```

Comment:
are both `setuptools` and `pip` *runtime* dependencies of any of the ones included below?

since you're using the system Python here, you're also at the mercy of the system Python to provide what's required to install Python packages...